test/new-e2e/tests/windows/common/user.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package common
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+)
+
+// JoinUserDomain joins a user and domain into a single string
+func JoinUserDomain(user string, domain string) string {
```

Comment:
At first glance, I thought this function might actually join a user to a an AD domain or something as opposed to concatenating the strings

Evidently, `domain\\user` is referred to as a "[Down-Level Logon Name](https://learn.microsoft.com/en-us/windows/win32/secauthn/user-name-formats#down-level-logon-name)", so maybe we could call this something like:

`MakeDownLevelLogonName` - if we did, probably worth linking the MSDN article because that isn't the most intuitive name on Microsoft's part 😅